frontend/src/components/payment-status/StatusIndicator.tsx

```
@@ -0,0 +1,133 @@
+            <Check className="h-3 w-3" aria-hidden="true" />
+          ) : (
+            <Clock className="h-3 w-3" aria-hidden="true" />
+          )}
+          <span className="text-xs font-medium">
+            {isPaid ? 'Paid' : 'Pending'}
+          </span>
+        </span>
+      </Badge>
+
+      {formattedTimestamp && (
+        <span className="text-xs text-gray-500" aria-label={formattedTimestamp}>
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)